test/device/math.jl

```diff
@@ -0,0 +1,41 @@
+@testset "Math Intrinsics" begin
+    for intr in AMDGPUnative.MATH_INTRINSICS
+        jlintr = intr.jlname
+        if intr.isbroken || !(isdefined(Base, jlintr) || isdefined(SpecialFunctions, jlintr))
+            @test_skip "$jlintr()"
+            continue
+        end
+        modname = (isdefined(Base, jlintr) ? Base : SpecialFunctions)
+        # FIXME: Handle all input and output args
+        T = intr.inp_args[1]
+        intr_kern = Symbol("intr_$(jlintr)_$T")
+        @eval begin
+            function $intr_kern(a)
+                i = threadIdx().x
+                a[i] = AMDGPUnative.$jlintr(a[i])
+                return nothing
+            end
+            dims = (8,)
+            a = rand($T, dims)
+            if $(QuoteNode(jlintr)) == :acosh
+                a .+= one($T)
+            end
+            d_a = HSAArray(a)
+            len = prod(dims)
+
+            if DEBUG
```

Comment:
Maybe use `@debug begin ... end`

Reply:
This seems to expect a `String` at the end, but I don't need to return one. Any ideas?

Reply:
`""`